internal/ingestion/pipeline/pipeline_test.go

```
@@ -455,6 +455,119 @@ func TestPipelineRunResumableCrossRunResume(t *testing.T) {
+		func(_ string, _ map[string]any) (runtime.Component, error) { return guard, nil },
+		runtime.Metadata{Version: "1.0.0"})
+	runtime.MustRegister(nameB, runtime.CategoryIngestion,
+		func(_ string, _ map[string]any) (runtime.Component, error) { return termStage, nil },
+		runtime.Metadata{Version: "1.0.0"})
+
+	const taskID = "task-orphan-interrupt"
+
+	store := newMemCheckpointStore()
+	mr := miniredis.RunT(t)
+	client := redis.NewClient(&redis.Options{Addr: mr.Addr()})
+	t.Cleanup(func() { client.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `client.Close` is not checked

(errcheck)